kubectl-plugin/src/operations.rs

```diff
@@ -9,6 +9,8 @@ pub(crate) enum Operations {
     Get(GetResources),
     /// 'Scale' resources.
     Scale(ScaleResources),
+    /// Version of the plugin
+    Version,
```

Comment:
This is a bit confusing now, we have a -V and a version?

Reply:
We don't have a `-V` option. It's use is only `kubectl-mayastor version`.

Reply:
You are correct about the `-V` option. As we discussed, this has now been fixed up.